### ISSUE TITLE
warning: The iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99.

### ISSUE THREAD

**Issue by dzoba** [react-native-market/firebase-authentication-kit#4]
I'm getting a lot of errors like this. I can't get the iOS project to run.  I've tried setting to the legacy build system, as stack overflow suggests, to no avail.

```Build system information
warning: The iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'BoringSSL-GRPC')

Build system information
warning: The iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'BVLinearGradient')

Build system information
warning: The iOS Simulator deployment target is set to 4.3, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'nanopb')

Build system information
warning: The iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'GTMSessionFetcher')

Build system information
warning: The iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'Protobuf')

Build system information
warning: The iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'gRPC-C++-gRPCCertificates-Cpp')

Build system information
warning: The iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'gRPC-Core')

Build system information
warning: The iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'GoogleSignIn')

Build system information
warning: The iOS Simulator deployment target is set to 5.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'leveldb-library')

Build system information
warning: The iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'gRPC-C++')

Build system information
warning: The iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'RNScreens')

Build system information
warning: The iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'react-native-fbsdk')```
![image](https://user-images.githubusercontent.com/857664/66249928-f300d100-e6ef-11e9-94f0-aede4eec4e0f.png)

**Comment by dzoba** [comment]
I seem to have gotten around this by adding 

```post_install do |pi|
    pi.pods_project.targets.each do |t|
        t.build_configurations.each do |config|
            config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '9.0'
        end
    end
end```

To /ios/podfile


Reference: https://github.com/CocoaPods/CocoaPods/issues/7314#issuecomment-355905199

**Comment by dzoba** [comment]
Ok next error solved.  I was getting something like

`Target 'FirebaseAuthenticationKit' (project 'FirebaseAuthenticationKit') has copy command from

I had to go to the Targets for the project, then the Build Phases then Copy Bundle Resources, and remove one of the GoogleService-info.plist lines.

Got a new error now

**Comment by dzoba** [comment]
Ok new error was because there were two GoogleService-info.plist lines in the Build Phases, and seems like I deleted the wrong one. One of them had small gray text to it that indicated where the file was kept.  That is the one to delete.

**Comment by Salakar** [comment]
@dzoba thanks for sticking through this and figuring things out (I've sent you a partial refund for your troubles 🙃).

I've published a new version (0.0.02) of the product, if you can; please try it out - you can download it in your account purchases section on the website. Thanks
